### PR TITLE
feat(runtimes): runtimes in KDL server

### LINKED ISSUE #699: Add runtimes in KDL
https://github.com/konstellation-io/konstellation.io/discussions/82

### PR DESCRIPTION
**Pull request by alejandrogr**
Closes #699

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@ coverage/
 helm/kdl-server/charts/
 node_modules
 backup/backup-pod.yaml
+main
```
Comment (aacecandev) [inline .gitignore:9]: blank line

File: app/api/.env.dev [hunk 2]
```diff
@@ -0,0 +1,33 @@
+GITEA_ADMIN_PASSWORD=a123456
```
Comment (aacecandev) [inline app/api/.env.dev:1]: envs should be defined in the docs instead here

File: jupyter-gpu/README.md [hunk 3]
```diff
@@ -1,41 +1,5 @@
-# Jupyter Lab GPU
+## APT Packages
```
Comment (aacecandev) [inline jupyter-gpu/README.md:1]: This docs should be improved (later) including Kernel Enterprise Gateway use manuals

File: runtime-jupyter/deployment.yml [hunk 4]
```diff
@@ -0,0 +1,64 @@
+apiVersion: apps/v1
```
Comment (aacecandev) [inline runtime-jupyter/deployment.yml:1]: Is this still needed?

File: user-tools-operator/helm-charts/usertools/values.yaml [hunk 5]
```diff
@@ -40,8 +41,18 @@ username: user.name
 
 usernameSlug: user-name
 
+kubeconfig:
+  externalServerUrl: https://192.168.0.21:16443
```
Comment (aacecandev) [inline user-tools-operator/helm-charts/usertools/values.yaml:45]: I'm not sure if it is a good idea to not have an empty string here

Also, values.yml urge for docs (table of required values + defaults)

File: app/api/.env.dev (hunk 2 above)
Comment (alejandrogr) [reply app/api/.env.dev:1]: fixed in 86a793abc

File: .gitignore (hunk 1 above)
Comment (alejandrogr) [reply .gitignore]: fixed in 86a793abc

File: app/api/.env.dev (hunk 2 above)
Comment (alejandrogr) [reply app/api/.env.dev:1]: So your proposal is to remove the `.env.dev` file and document all the environment variables needed in the `README.md`?

File: helm/kdl-server/values.yaml [hunk 6]
```diff
@@ -222,6 +275,12 @@ projectOperator:
       tag: v2
       pullPolicy: IfNotPresent
 
+# vscodeRuntime:
```
[the remark this answers is not in the document]
Comment (alejandrogr) [reply helm/kdl-server/values.yaml]: Yes, this was moved to the usertools chart. Fixed in 8274d1741

File: runtime-jupyter/deployment.yml (hunk 4 above)
Comment (alejandrogr) [reply runtime-jupyter/deployment.yml:1]: I think we dont need it. It was here from the first PoC we did with the runtimes. @alpiquero ?

Comment (alpiquero) [reply runtime-jupyter/deployment.yml]: Solved in 230d911b9e11b81c8cd060ccf0483de015463e1b

File: app/api/.env.dev (hunk 2 above)
Comment (aacecandev) [reply app/api/.env.dev:1]: I wasn't aware that variables were already explained elsewhere, so I'm resolving this discussion

File: app/ui/mock-server/src/mocks/meMock.js [hunk 7]
```diff
@@ -9,6 +9,7 @@ const me = {
   areToolsActive: true,
```
Comment (JYisus) [inline app/ui/mock-server/src/mocks/meMock.js:9]: @alejandrogr We should check if this field can be deleted

**Review by alpiquero — APPROVED** [review]

**Pull request merged**